configs/isnet/isnet_res50_8xb2-160k_ade20k-512x512.py

```diff
@@ -0,0 +1,72 @@
+_base_ = [
+    '../_base_/datasets/cityscapes.py', '../_base_/default_runtime.py',
```

Comment:
The dataset name is not matched with the config name. You might follow out [config doc](https://github.com/open-mmlab/mmsegmentation/blob/dev-1.x/docs/en/user_guides/1_config.md#config-name-style) to correct the config name.